atomics/T1068/T1068.yaml

```
@@ -0,0 +1,61 @@
+attack_technique: T1068
+display_name: 'Exploitation for Privilege Escalation'
+atomic_tests:
+- name: noPac
```

```suggestion
- name: SAM Name impersonation (noPac tool)
```

Or "sAMAccountName Impersonation", or any other that you prefer, but "noPac" only was a bit too short to precisely reference the attack ;)